indico/web/forms/fields/files.py

```diff
@@ -74,4 +75,39 @@ def process_formdata(self, valuelist):
             self.data = valuelist[0]
 
     def _value(self):
-        return self.get_metadata(self) if self.get_metadata else None
+        return None
+
+
+def get_file_metadata(file_):
```

Comment:
`file` is the one builtin where i wouldn't care much about shadowing. it's completely useless anyway (i guess that's why they removed it in pytrhon 3)